README.md

```diff
@@ -71,6 +71,11 @@ This option will help you when you want to save the output onto a file and edit
 When your terminal's background color is something light (e.g white), default color preset might look too bright and not readable.
 If so, specify `--light-background` as a command line argument. kubecolor will use a color preset for light-backgrounded environment.
 
+* `--force-colors`
+
+Forces colored output highlighting even if stdout is not a TTY.
+When you want to have colors, for example on `kubecolor get pod | grep Running`, you can force it via `--force-colors` flag: `kubecolor --force-colors get pod | grep Running`
```

Comment:
Sorry but I applied some changes after you submitted this PR.
And I think now it works with `grep` . Can you try it on your side?

Reply:
It does not work with pipes (Ubuntu 20.04)
![image](https://user-images.githubusercontent.com/57521/95892279-f745dc80-0d8e-11eb-8907-41b7ac4ea301.png)

Reply:
![image](https://user-images.githubusercontent.com/57521/95892489-50ae0b80-0d8f-11eb-91fe-d6cc8b33a0a5.png)

Reply:
Hmm this looks strange because it's working on my laptop.
Are you using bash or zsh? And which version?

![image](https://user-images.githubusercontent.com/60682957/95929987-56900500-0e00-11eb-8efd-cf2904d72782.png)